### ISSUE TITLE
Small IDE persistent Setup Elixir Facet SDK warning banner

### ISSUE THREAD

**Issue by drl123** [KronicDeth/intellij-elixir#1008]
Running plugin v7.2.1 on RubyMine 2017.3.2 I'm now getting a yellow banner at the top that `Elixir Facet SDK is not defined`, but I do have Elixir defined in settings.  I don't get this message in IntelliJ CE.  I also did not get this warning before Rubymine updated both itself and the plugin the other day.  This happens on both a new project and old project started before the upgrade that previously worked fine. 

My Elixir was installed via Homebrew at `/usr/local/Cellar/elixir/1.5.1`, if that makes any difference.

Is there any way to silence this warning?  The rest of the plugin seems to work as before.  I just get the annoying yellow bar at the top.

**Comment by KronicDeth** [comment]
I would try clearing the Elixir settings and then [setting](https://github.com/KronicDeth/intellij-elixir/tree/v7.2.1#small-ides) them again following.

**Comment by drl123** [comment]
OK...that worked.  I had to select Elixir from under Languages & Frameworks...but NOT expand it and set the SDK from there.  The link in the warning message took me to the SDKs sub-menu item and both that and the Internal Erlang SDKs were set, but the un-expanded one was not.  I set that to Elixir 1.5.1 and all is well.

Thanks for the quick reply on a Sunday!

**Comment by zbigniewpekala** [comment]
I must say that
```
I had to select Elixir from under Languages & Frameworks...but NOT expand it and set the SDK from there. The link in the warning message took me to the SDKs sub-menu item and both that and the Internal Erlang SDKs were set, but the un-expanded one was not.
```
is totally counter intuitive. Just struggled with that for 2h...

This `Facet SDK is not defined` link should bring user to the correct menu.

**Comment by rak-phillip** [comment]
Ha! I'm so glad I stumbled across this. Thanks for the clarification, @drl123.

**Comment by andlrb** [comment]
Thanks for clarification @drl123, that was my issue too!

**Comment by jimjames99** [comment]
Still had me stumped today. Thanks @drl123 !

**Comment by Pheric** [comment]
Still an issue over a year later. Thanks @drl123

**Comment by drl123** [comment]
@Pheric  It's actually resolved with the latest versions of the IDE from JetBrains and the latest version of the plugin.  In fact, I just did it last night with a clean repo cloned from GitHub and could set the Elixir version with no problem. (I'm on RubyMine 2019.1.1).  It no longer creates the .iml file with the Elixir Module in the root of the project that you had to edit.

The latest version of the plugin requires much less tweaking than previous ones.  It's a HUGE improvement.  The only thing I had to set was the SDK version and it now seems to retain the version between projects.  I used to have to set it whenever opening a new project for the first time, even when I only had one version installed.  I don't even need to create a file type for `.html.exs` files anymore...the highlighting just works!  It also now supports espec for testing, which is great if you are used to using rspec in Rails.

PS...if still having problems on OSX, try a full clean install of the IDE.  After uninstalling, go into `~/Library` and remove the folder matching your IDE (all versions) in each of the following directories: `Caches`, `Preferences`,` Logs`.  I ran into issues with the latest IDE getting its settings messed up (totally unrelated to this) when it imported from a previous version/installation.  Just reinstalling alone didn't resolve it.  I had to clean out these files to get it to work...again, not related to this, but if still having issues, worth trying.

I can confirm it's working well on 3 different Mac's.
